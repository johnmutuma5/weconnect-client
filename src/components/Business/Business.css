.Business {
  -webkit-box-flex: 1;
      -ms-flex-positive: 1;
          flex-grow: 1;
  -webkit-box-shadow: 0px 0px 1px 0px #96858F;
          box-shadow: 0px 0px 1px 0px #96858F;
  border-radius: 3px;
  margin-bottom: 45px;
  background: #FDFFFF;
}
