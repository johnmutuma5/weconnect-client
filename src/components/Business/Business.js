import React from 'react';
import PropTypes from 'prop-types';
import BusinessMainContent from './BusinessMainContent/BusinessMainContent';
import BusinessQuickLinks from './BusinessQuickLinks/BusinessQuickLinks';

import './Business.css';

const business = (props) => (
    <div className="Business flex_col">
        <BusinessMainContent { ...props } />
        <BusinessQuickLinks />
    </div>
);

business.propTypes = {
    name: PropTypes.string.isRequired,
    category: PropTypes.string.isRequired,
    location: PropTypes.string.isRequired
}

export default business;
