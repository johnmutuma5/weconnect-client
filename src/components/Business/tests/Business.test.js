import Business from '../Business';

describe('<Business /> Component', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<Business
                                    name='business name'
                                    category='tech'
                                    location='a place' />);
        expect(wrapper).toMatchSnapshot();
    })
})
