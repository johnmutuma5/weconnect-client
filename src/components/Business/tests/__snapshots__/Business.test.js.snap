// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Business /> Component renders correctly 1`] = `
<div
  className="Business flex_col"
>
  <businessMainContent
    category="tech"
    location="a place"
    name="business name"
  />
  <businessQuickLinks />
</div>
`;
